data/tanitoluwam.json

```diff
@@ -8,7 +8,6 @@
       "url": "https://twitter.com/adenugatani",
       "icon": "FaTwitter"
     },
-    {
       "name": "Youtube",
       "url": "https://youtube.com/tanisjourney7489",
       "icon": "FaYoutube"
```

Comment:
Please do not modify anyone else's profiles, you can remove these changes from your PR as it will block your PR from getting merged